cmd/paths.go

```diff
@@ -45,21 +45,26 @@ func pathsGenCmd() *cobra.Command {
 				return fmt.Errorf("chains need to be configured before paths to them can be added: %w", err)
 			}
 
+			port, err := cmd.Flags().GetString(flagPort)
```

Comment:
I would like to see this as an argument not a flag tbh, also users should pass in src-port and dst-port

Reply:
Sure. Change it to args.